cmd/kubeadm/app/cmd/phases/reset/removeetcdmember.go

```diff
@@ -56,14 +59,31 @@ func runRemoveETCDMemberPhase(c workflow.RunData) error {
 	etcdManifestPath := filepath.Join(kubeadmconstants.KubernetesDir, kubeadmconstants.ManifestsSubDirName, "etcd.yaml")
 	etcdDataDir, err := getEtcdDataDir(etcdManifestPath, cfg)
 	if err == nil {
-		r.AddDirsToClean(etcdDataDir)
 		if cfg != nil {
 			if !r.DryRun() {
-				if err := etcdphase.RemoveStackedEtcdMemberFromCluster(r.Client(), cfg); err != nil {
+				err := etcdphase.RemoveStackedEtcdMemberFromCluster(r.Client(), cfg)
+				if err != nil {
 					klog.Warningf("[reset] Failed to remove etcd member: %v, please manually remove this etcd member using etcdctl", err)
+				} else {
+					if err := CleanDir(etcdDataDir); err != nil {
+						klog.Warningf("[reset] Failed to delete contents of %q directory: %v", etcdDataDir, err)
+					} else {
+						fmt.Printf("[reset] Deleted contents of etcd data directories: %v\n", etcdDataDir)
+					}
 				}
 			} else {
 				fmt.Println("[reset] Would remove the etcd member on this node from the etcd cluster")
+				fmt.Printf("[reset] Would delete contents of etcd data directories: %v\n", etcdDataDir)
+			}
+		}
+		// This could happen if the phase `cleanup-node` is run before the `remove-etcd-member`.
+		// Cleanup the data in the etcd data dir to avoid some stale files which might cause the failure to build cluster in the next time.
+		empty, _ := IsDirEmpty(etcdDataDir)
+		if !empty && !r.DryRun() {
+			if err := CleanDir(etcdDataDir); err != nil {
+				klog.Warningf("[reset] Failed to delete contents of %q directory: %v", etcdDataDir, err)
+			} else {
+				fmt.Printf("[reset] Deleted contents of etcd data directories: %v\n", etcdDataDir)
```

Comment:
we can make these consistent

```suggestion
				klog.Warningf("[reset] Failed to delete contents of the etcd data directory: %v", etcdDataDir, err)
			} else {
				fmt.Printf("[reset] Deleted contents of the etcd data directory: %v\n", etcdDataDir)
```